python/examples/custom_incremental_pipeline.py

```diff
@@ -389,10 +389,10 @@ def main(
         logging.fatal(f"Image path does not exist: {image_path}")
     output_path.mkdir(exist_ok=True, parents=True)
     reconstruction_manager = pycolmap.ReconstructionManager()
-    if input_path is not None and input_path != "":
-        reconstruction_manager.read(input_path)
+    if input_path:
+        reconstruction_manager.read(str(input_path))
 
-    with pycolmap.Database.open(database_path) as database:
+    with pycolmap.Database.open(str(database_path)) as database:
```

Comment:
The cast to str really isn't nice, I think that we should fix the stubs instead - let me look into this.